views-core/src/main/java/io/micronaut/views/ViewsRenderer.java

```diff
@@ -37,39 +38,44 @@
     String EXTENSION_SEPARATOR = ".";
 
     /**
-     * @param viewName view name to be render
+     * @param viewName view name to be rendered
      * @param data     response body to render it with a view
      * @return A writable where the view will be written to.
      */
-    @NonNull Writable render(@NonNull String viewName, @Nullable Object data);
+    @NonNull
+    Writable render(@NonNull String viewName, @Nullable Object data);
 
     /**
-     * @param viewName view name to be render
+     * @param viewName view name to be rendered
      * @param data     response body to render it with a view
      * @param request  HTTP request
      * @return A writable where the view will be written to.
      */
-    default @NonNull Writable render(@NonNull String viewName, @Nullable Object data,
-            @NonNull HttpRequest<?> request) {
+    default @NonNull
+    Writable render(@NonNull String viewName, @Nullable Object data,
+                    @NonNull HttpRequest<?> request) {
         return render(viewName, data);
     }
 
     /**
-     * @param viewName view name to be render
+     * @param viewName view name to be rendered
      * @return true if a template can be found for the supplied view name.
      */
     boolean exists(@NonNull String viewName);
 
     /**
-     * Creates a view model for the given data.
+     * Converts the object being passed in into a {@code Map<String, Object>} if not already one.
+     *
      * @param data The data
      * @return The model
      */
-    default @NonNull Map<String, Object> modelOf(@Nullable Object data) {
+    default @NonNull
+    Map<String, Object> modelOf(@Nullable Object data) {
```

Comment:
I contemplated removing this method but I decided for keeping it to minimize the backwards breaking changes